### ISSUE TITLE
doxit jade problems

### ISSUE THREAD

**Issue by acornejo** [shannonmoeller-archive/doxit#2]
I tried to use doxit today, but there seem to be some issues with jade:

``` bash
doxit myfile.js                                                                                                                                                          
Warning: missing space before text for line 9 of jade file.
Warning: missing space before text for line 16 of jade file.

/usr/local/lib/node_modules/doxit/node_modules/jade/lib/runtime.js:200
  throw err;
        ^
Error: /usr/local/lib/node_modules/doxit/lib/../templates/default/index.jade:18
    16|       content: " ";
    17|       display: block;
  > 18|       position: absolute;
    19|       right: 0;
    20|       top: 0;
    21|       z-index: -1;

unexpected text ;

    at Object.Lexer.fail (/usr/local/lib/node_modules/doxit/node_modules/jade/lib/lexer.js:783:11)
    at Object.Lexer.next (/usr/local/lib/node_modules/doxit/node_modules/jade/lib/lexer.js:841:15)
    at Object.Lexer.lookahead (/usr/local/lib/node_modules/doxit/node_modules/jade/lib/lexer.js:111:46)
    at Parser.lookahead (/usr/local/lib/node_modules/doxit/node_modules/jade/lib/parser.js:109:23)
    at Parser.parseTag (/usr/local/lib/node_modules/doxit/node_modules/jade/lib/parser.js:671:25)
    at Parser.parseExpr (/usr/local/lib/node_modules/doxit/node_modules/jade/lib/parser.js:197:21)
    at Parser.tag (/usr/local/lib/node_modules/doxit/node_modules/jade/lib/parser.js:742:29)
    at Parser.parseTag (/usr/local/lib/node_modules/doxit/node_modules/jade/lib/parser.js:678:17)
    at Parser.parseExpr (/usr/local/lib/node_modules/doxit/node_modules/jade/lib/parser.js:197:21)
    at Parser.block (/usr/local/lib/node_modules/doxit/node_modules/jade/lib/parser.js:644:25)
```

I suspect, this might have to do with a newer version of jade? (However, you should be able to lock to a jade version on package.json).

**Comment by acornejo** [comment]
I've confirmed the break is to the version of jade. Fixing an older version of jade in the package.json file fixed the problem.

**Comment by goliatone** [comment]
Still happening, 0.28.0 works...

**Comment by shannonmoeller** [comment]
Fixed as of 1.2.1, published this morning.
